connector-email/src/main/resources/application.properties

```
@@ -0,0 +1,51 @@
+notifications.connector.kafka.incoming.group-id=notifications-connector-email
+notifications.connector.kafka.incoming.topic=${mp.messaging.tocamel.topic}
+notifications.connector.kafka.outgoing.topic=${mp.messaging.fromcamel.topic}
+notifications.connector.name=email_subscription
+notifications.connector.redelivery.counter-name=camel.email.retry.counter
+
+quarkus.http.port=9003
+
+quarkus.log.cloudwatch.enabled=false
+quarkus.log.cloudwatch.level=INFO
+quarkus.log.cloudwatch.log-stream-name=notifications-connector-email
+
+quarkus.log.sentry=false
+quarkus.log.sentry.in-app-packages=com.redhat.cloud.notifications
+
+quarkus.kafka.devservices.port=9092
+
+camel.component.kafka.brokers=localhost:9092
+camel.component.kafka.sasl-jaas-config=""
+camel.component.kafka.sasl-mechanism=GSSAPI
+camel.component.kafka.security-protocol=PLAINTEXT
+camel.component.kafka.ssl-truststore-location=
+camel.component.kafka.ssl-truststore-type=JKS
+camel.component.kafka.poll-on-error=RECONNECT
+camel.component.kafka.retries=3
+camel.component.kafka.retry-backoff-ms=200
+
+camel.context.name=notifications-connector-email
+
+mp.messaging.tocamel.topic=platform.notifications.tocamel
+mp.messaging.fromcamel.topic=platform.notifications.fromcamel
+
+%test.quarkus.log.category."com.redhat.cloud.notifications".level=DEBUG
+
+# The "no reply" address used to set as the sender in the emails the connector
+# sends.
+notifications.connector.email.no-reply.address=noreply@redhat.com
+
+# Configuration of the various user providers that the connector supports.
+notifications.connector.user-provider.bop.api_token=changeme # matches the header name sent to BOP/MBOP.
+notifications.connector.user-provider.bop.client_id=changeme # matches the header name sent to BOP/MBOP.
```

This needs to go away. That was only meant to explain my suggestion in the PR but not to be included in a commit :smiley: 
```suggestion
notifications.connector.user-provider.bop.api_token=changeme
notifications.connector.user-provider.bop.client_id=changeme
```

---

Oops! My bad. Gone!